### DISCUSSION TITLE
Update Wrapper 2.9.1.0 have bug.

### DISCUSSION THREAD

**Discussion by kxlins** [The-Wineskin-Project/WineskinServer#244] [General]
![2022-04-12317](https://user-images.githubusercontent.com/97193005/163427863-36e9750b-5ed6-40ea-a775-fdea97c656a4.png)

Click Browse, nothing happens

**Comment by Gcenx** [comment]
Was that with a newly created wrapper?

**Comment by kxlins** [comment]
yes.create new blank wrapper.

**Reply by Gcenx** [reply]
I’ll look at this over the weekend, for now manually set the desired path.

**Comment by kxlins** [comment]
OK, this is a viable solution.

**Comment by Gcenx** [comment]
Quickly looked over Wineskin.app sources and believe I’ve fixed the issue as it worked for me now so pushed an updated wrapper.

**Comment by kxlins** [comment]
Thank you!
